### PR TITLE
Alpha

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

Release latest alpha changes:
- MultiSig modify allows for requiredSignatures to be updated
- MultiSig proposer can reject their proposal
- Mark prior breaking change in instruction legs as breaking

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -44,9 +44,9 @@ jobs:
           mv coverage/clover.xml coverage/clover-${{matrix.shard}}.xml
           mv coverage/lcov.info coverage/lcov-${{matrix.shard}}.info
           mv coverage/coverage-final.json coverage/coverage-${{matrix.shard}}.json
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
-          name: coverage-artifacts
+          name: coverage-artifacts-${{ matrix.os }}-${{ strategy.job-index }}
```
Comment (polymath-eric) [inline .github/workflows/main.yml:49]: The job is resolving this to: `coverage-artifacts--1`

@prashantasdeveloper is `.os` the right property? Shouldn't it be `matrix.shard` ?

Comment (polymath-eric) [reply .github/workflows/main.yml:49]: PR: https://github.com/PolymeshAssociation/polymesh-sdk/pull/1294

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**